poetry.lock

```diff
@@ -621,7 +621,7 @@ bcrypt = ["bcrypt"]
 
 [[package]]
 name = "django-ansible-base"
-version = "2024.9.2.0.dev8+g5f718b4"
+version = "2024.9.12.0.dev4+ga44d6bf"
```

Comment:
I don't see a 9.12 tag

Reply:
This is auto generated by `poetry lock --no-update` command. The pyproject.toml has dependency `django-ansible-base = { git = "https://github.com/ansible/django-ansible-base.git", branch = "devel"`

Reply:
@AlanCoding should we lock the dab dependency to the latest release tag?

Reply:
@bzwei I don't think so, In the past we have it pinned but it has been more annoying that helpful, because DAB is in constant development, so to enable any new change we have to bump the version. So we finally decided to work from HEAD. 

cc @Dostonbek1 

I created https://github.com/ansible/eda-server/pull/1045 to help us to update the poetry.lock so we don't have to take care of it.